client/src/config.rs

```diff
@@ -0,0 +1,97 @@
+//! Module for client-related configuration and structs
+
+// FIXME
+#![allow(unused, missing_docs)]
+
+use core::str::FromStr;
+use std::{num::NonZeroU64, path::Path, time::Duration};
+
+use derive_more::Display;
+use eyre::Result;
+pub use iroha_config::base;
```

Comment:
we cannot have the whole of `iroha_config::base` given to the client

Reply:
Removed this re-export